src/components/BookSearch/BookList.js

```diff
@@ -0,0 +1,48 @@
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import "../styles/card-style.css";
+import { Rating } from '@mui/material';
+import "../styles/global-styles.css";
+
+export default function BookList({ books }) {
+
+    const renderedBooks = books.map((book) => {
+        return(
+            <Card
+                className="card rounded text-center border border-primary pb-0"
+                bg="dark"
+                text="light"
+                key={book.isbn13}>
+                    <div className="row g-5">
+                        <div className="col-md-4 my-0 pt-5">
```

Comment:
Jako że używamy react-bootstrap to powinniśmy też używać elementów z tej biblioteki czyli zamiast w class name zamiast row/col możesz użyć `<Row></Row>` oraz `<Col></Col>`
np. dla tej lini (17) zamiast tego wstawiłem `<Col md={4}>` i efekt był ten sam

Reply:
No ale czasem trzeba klasy niestety, np Card.Body nie przyjmuje mi parametru dla paddingu

Reply:
Ale gdzie nie trzeba klasy to poprawie